src/components/sbb-menu/sbb-menu.tsx

```diff
@@ -352,7 +398,25 @@ export class SbbMenu implements ComponentInterface {
               ref={(menuContentRef) => (this._menuContentElement = menuContentRef)}
               class="sbb-menu__content"
             >
-              <slot />
+              {this._actions ? (
+                [
+                  <ul class="sbb-menu-list">
```

Comment:
maybe provide a way to bring in an aria-label for the list, maybe listAccessibilityLabel?